core/src/main/java/dk/alexandra/fresco/logging/DefaultPerformancePrinter.java

```diff
@@ -0,0 +1,19 @@
+package dk.alexandra.fresco.logging;
+
+import java.util.Map.Entry;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DefaultPerformancePrinter implements PerformancePrinter {
+
+  public Logger log = LoggerFactory.getLogger(PerformanceLogger.class);
+  
+  @Override
+  public void printPerformanceLog(PerformanceLogger logger, int myId) {
+    String s = "Logger for "+this.getClass().getName()+": ";
```

Comment:
There is a refactoring error here. It should probably be`logger.getClass().getName()`. An alternative Approach would be to remove this line and change the last line to `LoggerFactory.getLogger(logger.getClass()).info(s)`. The latter might give nicer output.

Reply:
Also, please run checkstyle.